sheets/1/exercise-3.tex

```diff
@@ -5,6 +5,24 @@
 In jeder teilaufgabe wir ein bestimmter teil von Java erklärt und du sollst ein Beispiel aus Aufgabe 1 finden. In \textbf{[]} steht immer wie viele verschiedenen Beispiele du finden solltest.
 
 \begin{enumerate}
+    \item \textbf{[2]} Bevor wir uns mit dem Java Programmcode ansich beschäftigen, sollten wir noch Wissen was kommentare sind.
+    Weil es kommt vor das wir dinge in die Java datei schreiben wollen, einfach als Notiz oder auch als Erklärung, für andere Menschen. Da der Computer sie nicht lesen soll, müssen wir sie im Markieren.
+          Für kurze Kommentare kann man \lstinline[breaklines=false]{//} verwenden. Dammit markiert man den Rest der Zeile als Kommentar. Beachte das manche Programme, automatisch ein Zeilen umbruch einfügen wenn die Zeile zu lang wird.
+          Das ist aber meistens nur Darstellung und gilt nicht als ende der Zeile.
+          Alternativ kann man den kommentar auch mit \lstinline[breaklines=false]{/* */} umschließen. Dadurch kann man auch ein mehrzeiligen kommentar schreiben. Dabei ist es konnvention, jede Zeile mit einem \textcolor{javagreen}{\texttt{*}} zu beginnen.
+          \begin{lstlisting}[title=\textbf{Kommentar Beispiel}]
+// Ein Kurzer Kommentar 
+// Man kann übrigens am fehlen einer Zeilenummer gut erkennen, wenn ein Automatischer umbruch stattfindet.
+    
+class void not_even_Java //Auch an der Grünen färbung kann man in den Meisten Programmen den Kommentar erkennen.
+//Und in dieser PDf wir auch ein Pfeil als markierung für einen Automatischen Zeilen umbruch verwendet.
+/*
+ *  Allerdings sollte man das eher vermeiden.
+ * Für tatsächliche Mehrzeiligen Kommentare werden diese Blöcke empfohlen
+ * Auch hier kann natürlich automatisch umgebrochen werden.
+ */
+        \end{lstlisting}
+        Alle in < > geschriebenen Wörter sind Platzhalter für das was sie beschreiben. Jeder kann mit etwas anderem Ausgefüllt werden, auch wenn sie gleich heißen.
```

Comment:
```suggestion
        In dieser Aufgabe sind alle in < > geschriebenen Wörter Platzhalter für das was sie beschreiben. Jeder Platzhalter kann mit etwas anderem Ausgefüllt werden, auch wenn sie gleich heißen. Die Klammern <> müssen dabei mit ersetzt werden!
```
Diese allgemeine Anmerkung vielleicht ganz aus dem subtask Enumeration rausnehmen und davor (in einer Infobox) haben für bessere sichtbarkeit.